src/mock/mock-api.ts

```diff
@@ -1,7 +1,12 @@
 import fetchMock from 'fetch-mock';
 import { statistikkApiUrl } from '../statistikk/Statistikk';
-import { antallFormidlinger } from './testdata';
+import { hentAntallFormidlinger } from './testdata';
 
 fetchMock.config.fallbackToNetwork = true;
 
-fetchMock.get(`begin:${statistikkApiUrl}`, antallFormidlinger);
+fetchMock.get(`begin:${statistikkApiUrl}`, (url) => {
```

Comment:
Litt bedre mocking for å verifisere at app ikke lastes inn på nytt når prop (nav-kontor) endrer seg med ny container-arkitektur